docs/SRS-Volere/SRS.tex

```diff
@@ -430,9 +430,128 @@ \subsection{Specifying a Business Use Case (BUC)}
 
 \section{Business Data Model and Data Dictionary}
 \subsection{Business Data Model}
-\lips
+\begin{figure}[h]
+  \includegraphics[width=1\textwidth]{4G06A-data-model.png}
+  \caption{Data Model}
+\end{figure}
+
 \subsection{Data Dictionary}
-\lips
+\textbf{Classes}
+\begin{itemize}
+  \item  User
+    \begin{itemize}
+      \item This class represents a user of the portal
+      \item \textbf{Attributes:}
+        \begin{itemize}
+          \item userData: UserData -- It is the data related to the
```

Comment:
we got feedback about this on the first deliverable didn't we "Use Unicode characters. For example, you can use right dash (just the minus sign `-`, endash `–` or emdash `—`) instead using multiple `-`s."

Reply:
oh wait actually I think this is fine idk what that line of feedback meant which is why I commented, but I think this is fine